tests/testthat/_snaps/assert.md

```diff
@@ -243,6 +243,79 @@
       Error in `my_function()`:
       ! `my_arg$foo` must be a vector, not an environment.
 
+# list_check_all_size() works
+
+    Code
+      (expect_error(list_check_all_size(list(1:2, 1:3), 2)))
+    Output
+      <error/vctrs_error_assert_size>
+      Error:
+      ! `list(1:2, 1:3)[[2]]` must have size 2, not size 3.
+    Code
+      (expect_error(list_check_all_size(list(1:2, 1:3), 2, arg = "arg", call = call(
```

Comment:
By default I'd rather see a wrapper test than an explicit arg/call test. It's fine to have both of course.